roadmap/checklist.md

```diff
@@ -53,63 +53,63 @@
+|                 | openJDK       |           |            |        | ✅ openjdk-19   |          | ✅ openjdk-11 |        |         |        | ✅ openjdk-11 (bisheng & dragonwell & OpenJDK) | ✅ openjdk-8 |        |
+|                 | Nodejs        |           |            |        | ✅ nodejs-18.7.0 |          | ⚠️             |        |         |        | ✅ nodejs-v16.15.1 | ✅ nodejs-12.22.9 |  |
+|                 | Ocaml         |           |            |        | ✅ ocaml-4.13.1      |          |                |        |         |        | ✅ ocaml-4.12.0-3       |           |        | 
+|                 | golang        |           |            |        | ✅ golang-1.19     |          | ✅ golang-1.19 |        |         |        | ✅ golang-1.18.3       |           |        |
+|                 | erlang        |           |            |        | ✅ erlang-1:24.3.4.5      |          |        |        |         |        |        |           |        |
 | 构建工具        | obs           |           |            |        |        |          |        |        |         |        |        |           |        |
 |                 | osc           |           |            |        |        |          |        |        |         |        |        |           |        |
 |                 | obs-server    |           |            |        |        |          |        |        |         |        |        |           |        |
 | 容器            | runc          |           |            |        |        |          |        |        |         |        |        |           |        |
-|                 | docker        |           |            |        |        |          |        |        |         |        |        | ✅ docker.io-19.03.8 |        |
-| 桌面            | Xfce Desktop  |           |            |        |        |          |        |        |         |        |        |           |        | 
-|                 | KDE           |           |            |        |        |          |        |        |         |        |        |           |        |
-|                 | LXDE          |           |            |        |        |          |        |        |         |        |        |           |        |
-|                 | GNOME         |           |            |        |        |          |        |        |         |        |        |           |        |
+|                 | docker        |           |               |        |        |          |        |        |         |        |        | ✅ docker.io-19.03.8 |        |
+| 桌面            | Xfce Desktop  |           |            |        | ✅ xfce4-4.16       |          |        |        |         |        |        |           |        | 
+|                 | KDE           |           |            |        | ✅ kde-5:129       |          |        |        |         |        |        |           |        |
```

Comment:
Virtual package versions make little sense here, since we are trying to track the real version of desktop software suite. I would suggest to point to the real package versions with plasma-workspace, plasma-frameowkr, gnome-shell, etc.